specification/Overview.md

```diff
@@ -89,54 +173,89 @@ organised by data category (saveframe).
       LYS HZ%, ...). Notably N-terminal -NH3+ would be 'H%', as would
       N-terminal -NH2.  '\*' would only be
       used where necessary, mainly for 'H\*', 'C\*', '\*',  ... (all protons, all
-      carbons,  all atoms, etc.).
+      carbons,  all atoms, etc.), or for atom names that do not use digits for
+      distinguishing stereochemistry (e.g. DNA/RNA H5' and H5'', where the
+      two-atom set is designated H5'\*).
 
-    10. IUPAC pseudoatom names (ALA MB, SER QB, etc.) are NOT expanded into
+    11. IUPAC pseudoatom names (ALA MB, SER QB, etc.) are NOT expanded into
     sets of atoms. They are reserved for the
     original meaning, i.e. a geometric point positioned at the centroid with zero
     van der Waals radius. A restraint to ALA MB would be different from one to
-    ALA HB%, and it would be an error to confuse them. E.g. ALA MB and ALA HB%
-    can both appear in the same shift list, if both are needed e.g. in different
-    restraint lists.
+    ALA HB%, and it would be an error to confuse them. Note that a restraint to
+    HB% is calculated by r-6 averaging in most programs, whereas a restraints
+    to MB is not. The latter therefore needs to modify the given distance limits
+    to obtain the same result.
 
-    11. Pseudoatom/wildcard names, stereospecific or non-stereospecific atom
+    12. Pseudoatom/wildcard names, stereospecific or non-stereospecific atom
     names can coexist for the same atom in the same file, but the proper name
     must be used with its defined meaning throughout the NEF file. It follows
     that when renaming the atoms to stereospecific entities, e.g. once the
     coordinates are known, the program must also maintain the name previously
     used to preserve continuity.
 
+      The chemical shift list must contain all known relevant shifts, but it is
+      not necessary to give the same chemical shift twice. If e.g. a Ser HB%
+      shift is given, the HBx, HBy, HB2, and HB3 are known to be the same.
+      Similarly HB% or QB can be used without any obligation to give their
```

Comment:
I should like to get rid of it full stop - let alone in the shift list. But the agreement was that we would support it, so we would need to get a consensus on changing that. And if people are using QB internally, in assignment, restraints, or both, is it really possible, or smart, to get them to rewrite their names for deposition, with the additional confusion that gives?

Apart from which I think we need to close the current set of changes fast now, and then make further changes later.

Is there anybody out there who can say that they want NEF to support Q and M pseudoatoms in the shift list - which would close down the discussion for now?